### DISCUSSION TITLE
Timestamp vs Random ID

### DISCUSSION THREAD

**Discussion by LorenzBischof** [srid/neuron#461] [General]
I read somewhere that you previously switched from timestamp IDs to randomly generated ones. I am new to the Zettelkasten method and would prefer timestamps. Is there a reason you made this change?

**Comment by srid** [comment]
Date IDs were dropped in #368 which links to https://github.com/srid/neuron-template/issues/4 describing a potential problem with them.

One way to bring them back is via plugins, right after neuron gets a [plugin-based architeture](https://github.com/srid/neuron/discussions/462).
